packages/browser/package.json

```diff
@@ -43,6 +43,8 @@
   },
   "dependencies": {
     "@sentry-internal/browser-utils": "8.0.0-alpha.9",
+    "@sentry-internal/feedback-modal": "8.0.0-alpha.9",
```

Comment:
browser uses 8.0.0-alpha.9, but the new feedback stuff is 8.0.0-alpha.7

Reply:
oh yeah, i put this up before the new releases got cut. will update.